.github/workflows/main.yml

```diff
@@ -29,6 +29,10 @@ jobs:
           python-version: "3.8"
           cache: "pip"
           cache-dependency-path: requirements.*.txt
+      - name: Install opensafely
+        run: |
+          python3 -m pip install --quiet --upgrade opensafely
+          opensafely --version
```

Comment:
Installing opensafely-cli here means running `just test` locally will fail unless opensafely happens to be on the path.  job-server has this as a dev requirement and it's been absolutely fine to work with, would that work here too?  It also has the benefit of installing a pinned version and dependabot bumping that for you.